src/server/api/controllers/events.controller.js

```diff
@@ -68,10 +68,33 @@ const deleteEvent = async (userId, eventId) => {
     throw new Error('Unauthorized', 403);
   }
 };
+const editEvent = async (eventId, userId, updatedEvent) => {
+  const users = [];
+  await knex('users')
+    .select('users.id')
+    .join('user_roles', 'users.id', '=', 'user_roles.user_id')
+    .where('user_roles.role_id', 3)
+    .then((records) =>
+      records.forEach((record) => {
+        users.push(record.id);
+      }),
```

Comment:
This is imperative code. While it is working, you should know how to rewrite it in functional style with `map`.

Reply:
used map